### ISSUE TITLE
WCAG violation: missing accessible name for input field and textarea field

### ISSUE THREAD

**Issue by KjerstiSteien** [zazuko/Yasgui#39]
According til WCAG 4.1.2 (Level A): For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined. 

This is not possible for the input field and textarea field in this SPARQL GUI.
